cmake/Modules/FindGflags.cmake

```diff
@@ -0,0 +1,66 @@
+###############################################################################
+#
+# CMake script for finding the GFlags library.
+#
+# https://gflags.github.io/gflags/
+#
+# Redistribution and use is allowed according to the terms of the 3-clause BSD
+# license.
+#
+# Input variables:
```

Comment:
What about `Gflags_STATIC` which is used below? I'd say you need to document it here or drop that part of the code below.